tools/tools.go

```diff
@@ -0,0 +1,5 @@
+package tools
+
+import (
+	_ "k8s.io/code-generator"
```

Comment:
Can you add a purpose of this package is to include this code-generator for vendor?

Reply:
Yes, exactly. We don't use the package in the code base so I had to include it and force it to be included.